tasks/hyrax-metadata-updates/tests/test-search-failures.js

```diff
@@ -6,28 +6,53 @@ const nock = require('nock');
 const { RecordDoesNotExist } = require('@cumulus/errors');
 const rewire = require('rewire');
 const HyraxMetadataUpdate = rewire('../index');
+const { secretsManager } = require('@cumulus/aws-client/services');
+const {
+  randomId
+} = require('@cumulus/common/test-utils');
 
 const getEntryTitle = HyraxMetadataUpdate.__get__('getEntryTitle');
 
+const cmrPasswordSecret = randomId('cmrPassword');
+
 const event = {
   config: {
     cmr: {
       oauthProvider: 'earthdata',
       provider: 'GES_DISC',
       clientId: 'xxxxxx',
       username: 'xxxxxx',
-      passwordSecretName: 'xxxxx'
+      passwordSecretName: cmrPasswordSecret
     }
   },
   input: {}
 };
 
+test.before(async () => {
+  await secretsManager().createSecret({
+    Name: cmrPasswordSecret,
+    SecretString: randomId('cmrPasswordSecret')
+  }).promise();
+});
+
 test.beforeEach(() => {
   process.env.CMR_ENVIRONMENT = 'OPS';
+
+  nock('https://cmr.earthdata.nasa.gov')
```

Comment:
Not worth holding up review on: but if we're mocking this same endpoint for all of our tests, we could just do it in `test.before` instead of `test.beforeEach`